<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>354</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>database</string>
    </property>
    <addaction name="actionImport_data"/>
   </widget>
   <widget class="QMenu" name="menuSimulate">
    <property name="title">
     <string>simulate</string>
    </property>
   </widget>
   <addaction name="menuDatabase"/>
   <addaction name="menuSimulate"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="mainWindow.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>new</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="mainWindow.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="mainWindow.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>save as</string>
   </property>
  </action>
  <action name="actionImport_SBML">
   <property name="text">
    <string>import SBML</string>
   </property>
  </action>
  <action name="actionExport_SBML">
   <property name="text">
    <string>export SBML</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
  <action name="actionAnalysis">
   <property name="text">
    <string>analysis</string>
   </property>
  </action>
  <action name="actionOde">
   <property name="text">
    <string>ode</string>
   </property>
  </action>
  <action name="actionAnaly_sbml">
   <property name="text">
    <string>analysis sbml</string>
   </property>
  </action>
  <action name="actionImport_data">
   <property name="text">
    <string>import data</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="mainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
